docs/README.md

```diff
@@ -1,28 +1,152 @@
 
-Crossplane is an open source multicloud control plane. It introduces workload and resource abstractions on-top of existing managed services that enables a high degree of workload portability across cloud providers. A single crossplane enables the provisioning and full-lifecycle management of services and infrastructure across a wide range of providers, offerings, vendors, regions, and clusters. Crossplane offers a universal API for cloud computing, a workload scheduler, and a set of smart controllers that can automate work across clouds.
+Crossplane is an open source multicloud control plane to manage your
+cloud-native applications and infrastructure across environments, clusters,
+regions and clouds. It enables deployment of applications and managed services
+from `kubectl` with your choice of cloud or on-premise providers, including
+independent service offerings. 
 
-<h4 align="center"><img src="media/arch.png" alt="Crossplane" height="400"></h4>
+Crossplane can be installed into an existing Kubernetes cluster to add managed
+service provisioning or deployed as a dedicated control plane for multi-cluster
+management and workload scheduling.
 
-Crossplane presents a declarative management style API that covers a wide range of portable abstractions including databases, message queues, buckets, data pipelines, serverless, clusters, and many more coming. It’s based on the declarative resource model of the popular [Kubernetes](https://github.com/kubernetes/kubernetes) project, and applies many of the lessons learned in container orchestration to multicloud workload and resource orchestration.
+Crossplane enables the community to build and publish Stacks to add more clouds
+and cloud services to Crossplane with support for out-of-tree extensibility and
+independent release schedules. 
 
-Crossplane supports a clean separation of concerns between developers and administrators. Developers define workloads without having to worry about implementation details, environment constraints, and policies. Administrators can define environment specifics, and policies. The separation of concern leads to a higher degree of reusability and reduces complexity.
+<h4 align="center"><img src="media/crossplane-overview.png" alt="Crossplane"
+height="400"></h4>
 
-Crossplane includes a workload scheduler that can factor a number of criteria including capabilities, availability, reliability, cost, regions, and performance while deploying workloads and their resources. The scheduler works alongside specialized resource controllers to ensure policies set by administrators are honored.
+Crossplane has four main feature areas:
+1. Crossplane Services - provision managed services from kubectl.
+1. Crossplane Stacks - extend Crossplane with new functionality.
+1. Crossplane Workloads - define complete applications and schedule across
+   clusters, regions, and clouds.
+1. Crossplane Clusters - manage multiple Kubernetes clusters from a single
+   control plane.
```

Comment:
This one is a little interesting. In reality a cluster is just like any other managed resource as far as Crossplane is concerned. I can see why we might want to call it out explicitly because many folks probably won't think of spinning up clusters when they think about spinning up managed resources, but on the other hand we don't actually have any special functionality for clusters apart from the functionality that already falls under bullets 1 and 3. 🤔